nixos/modules/services/web-apps/tt-rss.nix

```diff
@@ -564,9 +566,14 @@ let
       "Z '${cfg.root}' 0755 ${cfg.user} tt_rss - -"
     ];
 
-    systemd.services.tt-rss =
-      {
+    systemd.services = {
+      phpfpm-tt-rss = mkIf (cfg.pool == "${poolName}") {
+        restartTriggers = [
+          cfg.root
```

Comment:
`restartTriggers` only works with store paths. (In reality, restart triggers only trigger a restart when their stringified constant changes). 

So I suggest `restartTriggers  = [ tt-rss-config tt-rss ];`